autorest/codegen/models/operation_group.py

```diff
@@ -91,20 +91,25 @@ def imports(self, async_mode: bool, is_python3_file: bool) -> FileImport:
             file_import.add_submodule_import(
                 ".._vendor", "raise_if_not_implemented", ImportType.LOCAL
             )
-        file_import.add_submodule_import(
-            "typing", "TypeVar", ImportType.STDLIB, TypingSection.CONDITIONAL
-        )
-        file_import.define_mypy_type("T", "TypeVar('T')")
-        type_value = "Optional[Callable[[PipelineResponse[HttpRequest, {}HttpResponse], T, Dict[str, Any]], Any]]"
-        file_import.define_mypy_type(
-            "ClsType", type_value.format(""), type_value.format("Async")
-        )
+        if self.has_nonabstract_operation:
```

Comment:
looks good! only nit I have is I think it's a little weird to have `has_nonabstract_operation` bc 99.9% of operation groups will and we don't want to make "abstract" more of a deal than it is (hopefully this makes sense. We can change this to `if all(o.abstract for o in self.operations): return file_import` so we don't need this property on operation group